scale/mesos_api/tasks.py

```diff
@@ -103,6 +104,7 @@ def _create_docker_task(task):
     arguments = task.command_arguments.split(" ")
     for argument in arguments:
         mesos_task.command.arguments.append(argument)
+    mesos_task.command.arguments.append(settings.CONFIG_URI)
```

Comment:
After talking with Derick earlier I don't believe this is where the CONFIG_URI needs to be assigned. Still working on this. I've found a couple of different examples, but they contradict each other. Http should work with this as well based on what I am seeing. 

One Marathon example of this: 
[mesos_task.command.arguments.append(settings.CONFIG_URI)](mesos_task.command.arguments.append(settings.CONFIG_URI))